### PR TITLE
Move Cosmos DB documentation for GA

### PR DESCRIPTION
**Pull request by DavidBoike**
Selecting a Persistence diagram goes from…

image

to…

image

### CODE REVIEW

**Comment by DavidBoike** [comment]
It's OK that the integrity tests are failing, it's just the checker for prerelease.txt. Before we merge this we will have published the final 1.0.0 package and updated the PR to remove the asterisk.

**Review by adamralph — APPROVED** [review]

File: persistence/selecting.md [hunk 1]
```diff
@@ -29,18 +29,15 @@ graph TD
...
-I --> |Other|L[SQL]
-I --> |RabbitMQ/Azure Service Bus|J[No persistence]
-I --> |Azure Storage Queues|K[Azure Table]
+I --> |MSMQ|L[SQL]
+I --> |All other transports|J[None required]
+B --> |MongoDB|M[MongoDB]
 B --> |Service Fabric|C[Service Fabric]
 B --> |RavenDB|E[RavenDB]
-B --> |MongoDB|M[MongoDB]
 B --> |Azure|Q{IaaS/PaaS ?}
```
Comment (adamralph) [inline persistence/selecting.md:37]: Small thing, but this looks slightly wonky:

image

```suggestion
B --> |Azure|Q{PaaS/IaaS ?}
```

File: persistence/selecting.md [hunk 2]
```diff
@@ -29,18 +29,15 @@ graph TD
 start((start)) --> A{Using sagas<br /> or outbox?}
 A --> |Yes|B{Additional<br />requirements?}
```
Comment (kentdr) [inline persistence/selecting.md:30]: Were we going to use some other language here besides Additional Requirements?

Comment (adamralph) [reply persistence/selecting.md:30]: It's nasty, but I think it's out of scope of our current work.

**Pull request merged**